lib/api/products.js

```diff
@@ -3,6 +3,7 @@ import orderBy from "lodash/orderBy";
 import _ from "lodash";
 import { Meteor } from "meteor/meteor";
 import { ReactiveDict } from "meteor/reactive-dict";
+import { Session } from "meteor/session";
```

Comment:
I'd prefer to avoid using Meteor session, we removed that dependency a while ago. If you need something reactive, our preference has been to use reaction var

Reply:
done.